engine/execution/computation/manager.go

```diff
@@ -41,7 +41,7 @@ const (
 	ReusableCadenceRuntimePoolSize = 1000
 )
 
-var uploadEnabled = true
+var uploadEnabled = false
 
 func SetUploaderEnabled(enabled bool) {
```

Comment:
I wonder if `computation manager` needs to know about upload at all. That seems not the responsibility of computation manager to me. 

Should we let another module to handle and decide whether and how to upload?

cc @m4ksio

Reply:
I opened a different PR to implement this: https://github.com/onflow/flow-go/pull/3676